src-v2/styles/_theme-variables.scss

```diff
@@ -0,0 +1,126 @@
+:root {
+  // used for elements where sizing is necessary (such as icons, avatar)
+  --str-chat__spacing-px: 1px;
+  // used for margins, padding
+  --str-chat__spacing-0: 0;
```

Comment:
This is based on spacing in Tailwind (https://tailwindcss.com/docs/customizing-spacing), but I'm note sure how to use this variable